src/Sarif/AnalyzeContextBase.cs

```diff
@@ -75,6 +77,8 @@ public virtual IEnumerable<IOption> GetOptions()
 
         public Regex CompiledGlobalFileDenyRegex { get; set; }
 
+        public TraceEventSession TraceEventSession { get; set; }
```

Comment:
>TraceEventSession [](http://example.com/codeflow?start=15&length=17)

Is this a standard class or something that we are adding? I think the session is called "Event Trace Session (ets)", so if this is a class we are creating, maybe worth renaming to align with the ets naming.

Reply:
Not our class! I agree it's odd how the word order switches in various contexts. :)